install-dev/data/xml/hook.xml

```diff
@@ -138,8 +138,8 @@
     <hook id="actionOrderReturn">
       <name>actionOrderReturn</name><title>Returned product</title><description>This hook is displayed when a customer returns a product </description>
     </hook>
-    <hook id="displayProductButtons">
-      <name>displayProductButtons</name><title>Product page actions</title><description>This hook adds new action buttons on the product page</description>
+    <hook id="displayProductExtra">
```

Comment:
Need to 1.7.0.0 sql 

UPDATE `PREFIX_hook` SET `name` = 'displayProductExtra' WHERE `name` = 'displayProductButtons';

!